contrib/ci/AzureCI/hypershift-install-azure.yaml

```diff
@@ -0,0 +1,131 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: hypershift-ops
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: hypershift-ops
+  name: admin
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: "hypershift-ops:admin"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    namespace: hypershift-ops
+    name: admin
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: hypershift
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: hypershift-operator
+  namespace: hypershift
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+    - name: latest
+      from:
+        kind: DockerImage
+        name: quay.io/hypershift/hypershift-operator:latest
+      referencePolicy:
+        type: Local
+      importPolicy:
+        scheduled: true
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: hypershift
+  name: installer
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: hypershift-installer
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - apiGroup: ""
+    kind: ServiceAccount
+    namespace: hypershift
+    name: installer
+---
+apiVersion: apps/v1
```

Comment:
Sometimes in the past, I've messed up overriding the deployment of Hypershift operator, could you add something to recover the latest status of the version we want?. I was wondering if just applying this manifest will be enough?

Reply:
I think just applying this manifest should be okay.